### PR TITLE
[FIX] mail: remove unexisting fields from condition

### PR DESCRIPTION
**Pull request by phenix-factory**

### CODE REVIEW

File: addons/mail/static/src/models/composer_view.js [hunk 1]
```diff
@@ -897,20 +897,14 @@ registerModel({
          * @returns {boolean}
          */
         _computeIsInDiscuss() {
-            return Boolean(
-                (this.threadView && (this.threadView.threadViewer.discuss || this.threadView.threadViewer.discussPublicView)) ||
-                (this.messageViewInEditing && (this.messageViewInEditing.discuss || this.messageViewInEditing.discussPublicView))
```
Comment (alexkuhn) [inline addons/mail/static/src/models/composer_view.js:902]: fix is `this.messageViewInEditing && this.messageViewInEditing.isInDiscuss`

File: addons/mail/static/src/models/composer_view.js [hunk 2]
```diff
@@ -897,20 +897,14 @@ registerModel({
...
         /**
          * @private
          * @returns {boolean}
          */
         _computeIsInChatWindow() {
-            return Boolean(
-                (this.threadView && this.threadView.threadViewer.chatWindow) ||
-                (this.messageViewInEditing && this.messageViewInEditing.chatWindow)
-            );
+            return Boolean(this.threadView && this.threadView.threadViewer.chatWindow);
```
Comment (alexkuhn) [inline addons/mail/static/src/models/composer_view.js:907]: same but with `isInChatWindow`

**Comment by phenix-factory** [comment]
@alexkuhn Done !

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**